playbooks/support/scheduling.md

```diff
@@ -1,74 +1,45 @@
 # Scheduling
 
-The [Engineering On-Call schedule][schedule] is configured in OpsGenie. Trading shifts (because of vacations,
-obligations, etc.) is encouraged as long as the schedule is kept up-to-date. Please address any scheduling issues
-as early as possible.
+The [Engineering On-Call schedule](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main) is configured in OpsGenie and works on a rotation basis. Each engineer will be on-call for one week per rotation. A rotation lasts for the period of time it takes for each participant to be on-call once, and will repeat itself indefinitely.
 
-A month before the next round of on-call is slated to begin (can check when the current round ends to verify this
-timeline), we being scheduling the next round.
+Trading shifts (because of vacations, obligations, etc.) are encouraged as long as the schedule is kept up-to-date. Please address any scheduling issues as early as possible.
```

Comment:
```suggestion
Trading shifts (because of vacations, obligations, etc.) is encouraged as long as the schedule is kept up-to-date. Please address any scheduling issues as early as possible.
```